engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -119,7 +119,7 @@ <h4 translate>Subscriptions</h4>
 
       <dl class="dl-horizontal dl-horizontal-left">
         <dt translate>System Purpose Status</dt>
-        <dd>
+        <dd ng-hide="simpleContentAccessEnabled">
```

Comment:
I think this same check needs to be done on the above <dt> since I can still see the "System Purpose Status" text

Reply:
Fixed